.gitlab-ci.yml

```diff
@@ -88,107 +49,56 @@ variables:
-#Deploy jobs for old k8s cluster (review apps)
-#Deploy feature branch to dev (review apps with custom BE and production DB)
-Review:WP-HX:
-    extends: .deploy-review
-    environment:
-        name: wp-hx-dev-ing
-        url: http://$CI_COMMIT_REF_SLUG.review.ensembl.org
-        on_stop: Stop_review
-    except:
-        - main
-        - dev
-        - update
-
-#Deploy special feature branch to dev (review apps with custom BE and internal DB)
-Review_db:WP-HX:
-    extends: .deploy-review
-    environment:
-        name: wp-hx-dev-ing
-        url: http://update.review.ensembl.org
-    after_script: #restart deployment with new env that points to internal track database
-        - sed -i "s#<BRANCH_NAME>#update#g" k8s/review/ingress.yaml
-        - kubectl apply -f k8s/review/ingress.yaml
-        - kubectl set env deployment/ensembl-track-api-update-deployment  
-          --from configmap/ensembl-track-api-review-configmap
-    only:
-        - update
-
-#Stop review appp
-Stop_review:
-    extends: .stop-review
-    environment:
-        name: wp-hx-dev-ing
-        action: stop
-    except:
-        - main
-        - dev
-        - update
-
-#Deploy jobs for new k8s cluster
-Dev:WP51:HL:
-    extends: .deploy-main
-    environment:
-        name: wp51-hl-development
-
-Internal:WP40:HL:
-    extends: .deploy-dev
-    environment:
-        name: wp40-hl-internal
-
-Staging:WP40:HL:
-    extends: .deploy-dev
-    environment:
-        name: wp40-hl-staging
-
-Prod:WP41:HX:
-    extends: .deploy-main
-    environment:
-        name: wp41-hx-prod
-    when: manual
-
-Prod:wp40:HL:
-    extends: .deploy-main
-    environment:
-        name: wp40-hl-prod
-    when: manual
-
-# .setup-review:
-#    stage: setup
-#    image: dockerhub.ebi.ac.uk/ensembl-web/deploy-tools:latest
-#    before_script:
-#        - ( if [ $(kubectl get ns | grep -c ${CI_COMMIT_REF_SLUG}) == "0" ]; then kubectl create namespace  ${CI_COMMIT_REF_SLUG} --dry-run=client -o yaml | kubectl apply -f -; fi)        
-#        - git clone https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests.git
-#        - git -C ensembl-k8s-manifests/ checkout k8s123-migration
-#        - cd ensembl-k8s-manifests/ensembl-web-track-api
-#    script:
-#        - sed -i "s#<HOST>#${CI_COMMIT_REF_SLUG}.review.ensembl.org#g" ingress.yaml
-#        - kubectl -n ${CI_COMMIT_REF_SLUG} apply -f service.yaml
-#        - kubectl -n ${CI_COMMIT_REF_SLUG} apply -f ingress.yaml
-       
-#    rules:
-#        - if: $CI_PIPELINE_SOURCE == "push" && 
-#              $CI_COMMIT_BEFORE_SHA == "0000000000000000000000000000000000000000" && 
-#              $CI_COMMIT_REF_NAME !~  "/^nodeploy\/.*$/"
-#          when: always
-
-#SetupReview:HL51:
-#    extends: .setup-review
-#    environment:
-#        name: wp51-hl-development
-#    rules:
-#        - if: '$CI_DEPLOY_FREEZE == null && $CI_COMMIT_BRANCH != "main" && $CI_COMMIT_BRANCH != "dev"'
-#          when: on_success
+    extends: .build-branch-rules
```

Comment:
This should be build regardless of which branch it is.

Reply:
Done